doc/source/whatsnew/v0.25.0.rst

```diff
@@ -729,6 +729,10 @@ Sparse
 - Bug in :class:`SparseDataFrame` when adding a column in which the length of values does not match length of index, ``AssertionError`` is raised instead of raising ``ValueError`` (:issue:`25484`)
 - Introduce a better error message in :meth:`Series.sparse.from_coo` so it returns a ``TypeError`` for inputs that are not coo matrices (:issue:`26554`)
 
+ExtensionArray
```

Comment:
its ok to make a new section, but I think we have just been putting EA things in other notes, will have to move them (either in this PR is ok) or a followup